mantidproject/mantid
# Fix workbench start on package version on Windows

**Description of work.**

Fixes startup on the package version of workbench on Windows.

The `launch_workbench.pyw` contained handwitten code that was not updated in #28699 and the Qt plugin path loading wasn't moved to main. This only affected the package version on Windows and no other platforms so slipped through testing.

**To test:**

On Windows:
* Build the branch and check workbench starts for a dev build
* Install the package and check it starts there.

Fixes #28710

*This does not require release notes* because **it is a regression.**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
